### ISSUE TITLE
attributes disappeared in customized search results

### ISSUE THREAD

**Issue by AJLinn** [ArctosDB/arctos#7433]
**Describe the bug**
A number of attributes that used to be available as search result columns in the customize screen seem to have disappeared.
I regularly use the following in my search results:

- value
- description
- verbatim attribute

All three of these are gone as options when I go into the customize search and results tool:

<img width="622" alt="Screenshot 2024-02-19 at 4 20 59 PM" src="https://github.com/ArctosDB/arctos/assets/17605945/83096f8a-b7c3-4511-9066-7ef820baa72d">

These are essential options as outputs for my collection.

**Desktop (please complete the following information):**
 - OS: Mac OS 14.3.1
 - Browser Firefox 
 - Version 115.7.0esr (64-bit)

**Priority**
High - preventing viewing essential data in my search outputs

**Comment by mkoo** [comment]
HI Angie, those are all attributes so they are including in that class of data. I think we need to have an Arctos-wide chat about fields in flat and thus available as separate fields. Bio and Cultural collections have different needs. 
But for now, include Attributes in your checklist of results and then use the Tools-->View/Download Attributes for a clean filter and view of all your attributes. 

I am adding to the Issues meeting agenda...

**Comment by AJLinn** [comment]
> include Attributes in your checklist of results and then use the Tools-->View/Download Attributes for a clean filter and view of all your attributes.

yeah, that is just not an ideal solution for the types of search results I need to see in relation to one another and sort by. 

> I am adding to the Issues meeting agenda...

Unfortunately I can't be at the Issues Meeting on the 7th due to an in-house meeting so I'd appreciate a chance to talk about this at another time, if possible. I agree, this is a cultural vs. biological collection need difference!

**Comment by dustymc** [comment]
>report

Maybe/probably, but that hadn't been put on the table. (I know I get tunnel vision when the request starts as one possible solution, any sort of help getting out of that is greatly appreciated, thanks!)


---------------


Comment from the nuked issue:

> downloaded into a sortable and tally-able CSV file 

<img width="254" alt="Screenshot 2024-06-03 at 16 25 38" src="https://github.com/ArctosDB/arctos/assets/5720791/1deaadfb-c3a2-455b-b698-f0625d6023ff">

There are several download options there, perhaps one of them is what you're looking for.

I have no objections to adding this to the cache, but there are big-picture sustainability concerns which I'm unsure how to address. There are a finite number of possible columns, backups and replication are affected, it's just a "forever expense" that should be avoided if it can be (and paid if it must be, probably). @ArctosDB/arctos-working-group-officers help????


> February 2024

@mkoo @Jegelewicz that shouldn't happen, I'm completely overwhelmed in Issues (and suspect I'm not alone in that), can we meet up (and invite anyone willing to help!) and see if we can find some way to get the actionable/functional/whatever-this-is Issues to the top of the pile? (As a start, maybe I should close all of https://github.com/ArctosDB/arctos/issues?q=is%3Aopen+is%3Aissue+milestone%3Amystery a week after they're flagged or something; I'm up for anything, other than continually getting lost in the weeds of 455 issues, most of which I don't know how to address....)

**Comment by mkoo** [comment]
Seems like there are two methods to retrieve this info-- please file an issue if you would like something else (maybe a report or something?) Otherwise this issue is stale and has no action items
